src/utils/googleApi.js

```diff
@@ -0,0 +1,13 @@
+import { request } from "./api";
+
+const getPages = () => {
+  return request("/api/google")
+    .then((data) => {
+      return data.urls;
+    })
+    .catch((err) => {
+      console.error(`An error occurred: ${err}`);
```

Comment:
* You should **not** use `catch` in `Api` because you need to `catch` possible errors at the very **end** of any request, and this end is in (`App.js`) after all `then` blocks. And in real projects the `catch` blocks usually open error popups and so on.